core/cont/src/TClassTable.cxx

```diff
@@ -538,7 +592,8 @@ TClassRec *TClassTable::FindElementImpl(const char *cname, Bool_t insert)
 
 TClassRec *TClassTable::FindElement(const char *cname, Bool_t insert)
 {
-   if (!CheckClassTableInit()) return nullptr;
+   if (!CheckClassTableInit())
+      return nullptr;
```

Comment:
No lock required in this function?  :monocle_face:

Reply:
I think then it's useful to add a TODO comment in the code that mentions that the function is obsolete and should be removed.

Reply:
It is removed as part of an update to this PR.